packages/codefixes/src/typescript-codefix-collection.ts

```diff
@@ -16,7 +16,52 @@ import type { CodeFixCollection, DiagnosticWithContext } from './types';
 export class TypescriptCodeFixCollection implements CodeFixCollection {
   hasPrettier: boolean | undefined;
   prettierConfigs: PrettierOptions | undefined;
-  getFixForDiagnostic(diagnostic: DiagnosticWithContext): CodeFixAction | undefined {
+
+  // List of safe to apply code fix names
+  // @see ./helpers/available-fix-names.ts for the list of available fixNames
```

Comment:
similar comment - after we move `available-fix-names.ts` out of src to docs or even README.md link it here

Reply:
Moved to .md